RELEASE-NOTES.md

```diff
@@ -1,5 +1,10 @@
 ## RELEASE NOTES
 
+### Version 2.7.1 - December 24, 2018
+
+**Bugs**
+- **Loader** - Fixed a bug with the elastic loader applying the wrong color **Thanks [@prudho](https://github.com/prudho)** [#325](https://github.com/fomantic/Fomantic-UI/pull/325)
```

Comment:
It was fixed by me rather than prudho :wink:

Reply:
Fixed, sorry 😉